step.yml

```diff
@@ -109,8 +109,6 @@ inputs:
       description: |
           The path where build artifacts will be saved to if the **Wait for builds** input is set to `true`. 
       
-          Please note that the path must be an actual, existing path, ending with `/`. The `BITRISE_DEPLOY_DIR` Environment Variable is NOT a valid value. 
-      
```

Comment:
1. Actual, existing path: this PR makes sure that the path exists
2. Ending with `/`: it's no longer relevant, a previous PR added `filepath.Join()` calls to handle this automatically
3. `BITRISE_DEPLOY_DIR`: We never enforced this restriction and it seems to work fine. It's also the most convenient folder to use as artifact path.